src/esp/assets/ResourceManager.h

```diff
@@ -84,6 +88,17 @@ class ResourceManager {
   /** @brief Convenience typedef for Importer class */
   using Importer = Mn::Trade::AbstractImporter;
 
+#ifdef ESP_BUILD_WITH_VHACD
+  struct VHACDParameters : VHACD::IVHACD::Parameters {
+    VHACDParameters() {
+      m_oclAcceleration = false;  // OCL Acceleration does not work on VHACD
+    }
+    ESP_SMART_POINTERS(VHACDParameters)
+  };
+
+  VHACD::IVHACD* interfaceVHACD;
```

Comment:
This could be a unique_ptr instead and then it would destruct itself when ResourceManager is destructed.

Reply:
Can't, the destructor is protected. See [my comment above](https://github.com/facebookresearch/habitat-sim/pull/1055#discussion_r579428913).